config/groups.yaml

```
@@ -25,3 +25,9 @@
     - "2509095"
     - "2553338"
   id: 189499
+  - name: "SSHiverMeTimbers"
+  members:
```

In a YAML object, the starts of the keys must be aligned:
```suggestion
  - name: "SSHiverMeTimbers"
    members:
```